string camera_topic
---
int8 banknote_amount
